script_library/deployment-actions-common.sh

```diff
@@ -22,6 +22,16 @@ function deploy_osh(){
     run_ansible ${socok8s_absolute_dir}/playbooks/generic-deploy_osh.yml
 }
 
+function deploy_ses_rook(){
+    echo "Deploying SES with rook"
```

Comment:
I would extend it with "... into kubernetes cluster" because that's the difference to the other SES deployment

Reply:
well, rook is a kubernetes orchestrator so there shouldnt be much confusion, but ok, will change it

Reply:
fixed